src/mon/MDSMonitor.cc

```diff
@@ -1581,6 +1582,12 @@ int MDSMonitor::management_command(
       return -ENOENT;
     }
 
+    int64_t metadata_num_objects = mon->pgmon()->pg_map.pg_pool_sum[metadata].stats.sum.num_objects;
```

Comment:
It has just occurred to me that we will sometimes want to permit this, in order to enable people to do an "fs new", "fs reset" type procedure -- that is permitted for people who make the mistake of doing an "fs rm" but still have all their objects.  Could you add a --force flag here?